frame/vesting/src/tests.rs

```diff
@@ -96,50 +96,41 @@ fn check_vesting_status() {
 	});
 }
 
-#[test]
+#[test]	
 fn check_vesting_status_for_multi_schedule_account() {
-	ExtBuilder::default().existential_deposit(ED).build().execute_with(|| {
+	ExtBuilder::default().existential_deposit(ED).add_vest(2, ED * 10, ED, 0)
```

Comment:
Misses a schedule.

Reply:
I believe the missing schedules are added by default in the genesis config by the mock file. Should I use `add_vest` to add them instead?